web/modules/custom/rsvplist/src/EnablerService.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\rsvplist;
+
+use Drupal;
```

Comment:
Non-namespaced classes/interfaces/traits should not be referenced with use statements